projects/aca-playwright-shared/src/utils/general-utils.ts

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+export class GeneralUtils {
+
+
+  static random(): string {
+    return (Math.random() + 1).toString(36).substring(6);
```

Comment:
there's already a random util in another PR with different implementation?

Reply:
https://github.com/Alfresco/alfresco-content-app/pull/3335/files#diff-08391580e8082276eed287862446386e71d15e46a0778704448447b318f6019f

Reply:
@DenysVuika  it looks like mentioned PR is younger than this one so should I update to it anyway ?